www/js/recommendations.js

```diff
@@ -18,17 +18,20 @@ angular.module('emission.main.recommendations',['emission.services', 'emission.p
         var mode = JSON.stringify(result.data.mode);
         document.getElementById("message_1").innerHTML = name.replace(/\"/g, "");
         document.getElementById("mode_1").innerHTML = mode.replace(/\"/g, "");
-        // Eventually this will be a for loop with all the messages 
+        // Eventually this will be a for loop with all the messages
```

Comment:
I will accept this for now, but in general, you don't want to include lines that don't have any meaningful change.
It makes it harder to review.